lib/envs.js

```diff
@@ -0,0 +1,3 @@
+export const ENVS = {
+  a: 1,
+};
```

Comment:
this is...? 😅

Reply:
o I was making mistakes with env variables so I was going to store them all here and check for attributes. after that I would import it if I ever needed env variables